m0t1fs/linux_kernel/st/m0t1fs_client_inc.sh

```diff
@@ -114,12 +114,12 @@ unmount_and_clean()
 		# unmounting the client file system, from next mount,
 		# fids are generated from same baseline which results
 		# in failure of cob_create fops.
-		local ios_index=`expr $i + 1`
+		local ios_index=$((i + 1))
```

Comment:
Uniform $var is preferable. Also, it is CodAcy, not Codecy. Please fix in the ticket and in the pull requests.

Reply:
Thanks for the review.  We will correct the spelling. If we just remove expr and keep the $var for uniformity sake then we are getting another codacy  warning that is, "$/${} Is unnecessary on arithmetic variable". We also have a github issue for that codacy warning : https://github.com/Seagate/cortx-motr/issues/1610

Reply:
As the warning, "$/${} Is unnecessary on arithmetic variable" ( https://github.com/Seagate/cortx-motr/issues/1610 ) seems to be false positive we will disable it.

Reply:
Okay, will update the patch.

Reply:
Patch uipdated.